csg.js

```diff
@@ -5483,17 +5483,18 @@ for solid CAD anyway.
     // Each side is a line between 2 points
     var CAG = function() {
         this.sides = [];
-        this.isCanonicalized = false;
+        this.isCanonicalized = true;
```

Comment:
First, this makes the CAG constructor consistent the CSG constructor. Second, this change makes effects the speed of operations performed with a newly created CAG, i.e. Canonization is not required.

I haven't seen any side effects. And it may be possible to remove this "state" with proper mathematics later.